package.json

```diff
@@ -24,6 +24,10 @@
     "test:fastboot": "ember fastboot:test",
     "test:node": "mocha node-tests --recursive"
   },
+  "now": {
+    "name": "ember-simple-auth",
+    "alias": "ember-simple-auth"
```

Comment:
Ah I didn't know about these options. Should be `"alias": "ember-simple-auth.now.sh"`